universal-login-sdk/lib/core/observers/PriceObserver.ts

```diff
@@ -0,0 +1,40 @@
+import {TokenDetails} from '@universal-login/commons';
+import ObserverRunner from './ObserverRunner';
+const cryptocompare = require('cryptocompare');
+
+export type ObservedCurrency = 'USD' | 'EUR' | 'BTC';
+
+export class PriceObserver extends ObserverRunner {
```

Comment:
Isn't PriceObserver the same thing as PriceOracle?

Reply:
It is. Thus, I wanted naming to be consistent.

Reply:
Shall we meget those two?